package.json

```diff
@@ -18,6 +18,7 @@
     "ical": "^0.5.0",
     "ical.js": "^1.2.2",
     "mathjs": "^3.2.1",
+    "purescript": "^0.11.6",
```

Comment:
why was this added?

Reply:
My bad, that was me trying to get the purescript issues to go away (until trm provided a fix). Accidentally deleting package.json was me (poorly) trying to stop this from being included.

Reply:
bahaha no wozzas